### PR TITLE
ownerscheck with labels

### PR DESCRIPTION
**Pull request by itayd**
Signed-off-by: Itay Donanhirsh <itay@bazoo.org>

Description:
Configure ownerscheck to work with labels.

Risk Level:
Low.

Testing:
In this PR.

Docs Changes:
None

Release Notes:
None

### CODE REVIEW

**Comment by itayd** [comment]
api shepherds, please ignore.

File: repokitteh.star [hunk 1]
```diff
@@ -5,8 +5,15 @@ use("github.com/repokitteh/modules/circleci.star", secret_token=get_secret('circ
...
-    ("envoyproxy/api-shepherds!", "api/"),
-    ("envoyproxy/udpa-wg", "api/udpa/"),
+    {
+      "owner": "envoyproxy/api-shepherds!",
+      "path": "api/",
+      "label": "api",
+    },
+    {
+      "owner": "envoyproxy/udpa-wg",
+      "path": "api/udpa/",
```
Comment (htuch) [inline repokitteh.star]: UDPA now lives in a separate repo, so you can drop this.

Comment (itayd) [reply repokitteh.star]: 👍

**Pull request merged**